### ISSUE TITLE
Release next version / merge pull requests

### ISSUE THREAD

**Issue by actra-gschuster** [angular-redux/form#49]
### This is a...

* [x] feature request
* [ ] bug report
* [ ] usage question

@clbond @SethDavenport @smithad15 @e-schultz 

Especially the fix for broken reactive forms is absolutely important.
PR #42 is hanging around since september...

**Comment by anodynos** [comment]
Unfortunately, this project looks abandoned... There's no activity for the past ~6 months, issues and PR have stalled... 

Has anyone made or is willing to be making a maintained fork?  Are Andrew @smithad15 and  @angular-redux guys willing to give access to some other maintainer? 

Regards.

**Comment by e-schultz** [comment]
Hi,

If there is someone that has interest in becoming a maintainer, happy to discuss that with them.

I'd really like to keep angular-redux + the supporting projects going, but having a hard time giving things the attention that they deserve right now.

If your interested in helping out, please get in touch -  evan@rangle.io

**Comment by smithad15** [comment]
Apologies for the extended radio silence on the library. Ran out of OSS time over the end of last year. Will be putting in more time cleaning up old issues / PRs in the coming days as well as getting a couple of new versions released around Angular 5 support. I also want to get in a serious rework of the base code and fold in some useful additions I have been able to make elsewhere to help ease the ability to merge Angular Forms and Redux. Worry not, this has not been abandoned 🙂
